ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -511,6 +511,18 @@ VAV_RD_CO2C:
   - RD
   - CO2C
 
+VAV_PDSCV_CO2C_HLSAFS_SRC:
```

Comment:
given the other comment string, i think PDSCV gets replaced with SD and we are good here. use MISSING translation values to deal with data that it does not send us.

Reply:
you're right sir, I didn't see this combination before hehe. Thanks, will do this.